src/World.cpp

```diff
@@ -558,7 +558,7 @@ void cWorld::Start(void)
 	if (!BadWorlds.empty())
 	{
 		const char * WorldName = m_WorldName.c_str();
-		LOGERROR("\n###### ERROR: %s is linked to the following nonexisting world/s:\n%s\n\nPlease edit %s/world.ini and fix this.\n\nNote that the server started enforcing proper world linkages recently. And people with older configs may naturally get this error. If you just want a working default config and don't mind losing this world, delete the folder \"%s\" and the server will receate one for you. Otherwise edit the world.ini file and fix the invalid linkages.\n######\n",
+		LOGERROR("\n###### ERROR: \"%s\" is linked to the following nonexisting world/s:\n%s\n\nPlease edit %s/world.ini and fix this.\n\nNote that the server started enforcing proper world linkages recently. And people with older configs may naturally get this error. If you just want a working default config and don't mind losing this world, delete the folder \"%s\" and the server will receate one for you. Otherwise edit the world.ini file and fix the invalid linkages.\n######\n",
```

Comment:
Ah, about this, can you remove the slightly random newlines and the number sign (hash)? It literally occurs nowhere else, even when the server crashes spectacularly (i.e. often).